test/extensions/filters/network/common/fuzz/network_readfilter_fuzz.proto

```diff
@@ -27,8 +27,12 @@ message Action {
   }
 }
 
+message Actions {
+  repeated Action actions = 1;
+}
+
 message FilterFuzzTestCase {
   // This is actually a protobuf type for the config of network filters.
   envoy.config.listener.v3.Filter config = 1;
-  repeated Action actions = 2;
+  Actions actions = 2;
```

Comment:
This change will probably need an update of the current corpus, otherwise everything will be rejected.

Reply:
Correct. That was more a quick fix, because there was not interface to mutating a repeated field. I changed to use a helper message, to which I move the actions, mutate them and go back.